db/post_migrate/20230320122645_delete_duplicated_seeded_prompts.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class DeleteDuplicatedSeededPrompts < ActiveRecord::Migration[7.0]
```

Comment:
Is a post migration the right place to do this? 🤔 Aren't the seeds done _way_ earlier?

Reply:
Ah, didn't know that. Will move it into a migration then.